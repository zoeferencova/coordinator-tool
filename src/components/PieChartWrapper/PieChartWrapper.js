import React, { Component } from 'react';
import PieChart from '../PieChart/PieChart';

export default class PieChartWrapper extends Component {
    componentDidMount() {
        this.setState({ chart: new PieChart(this.refs.chart) })
    }

    render() {
        return (
            <div ref="chart"></div>
        );
    }
}
