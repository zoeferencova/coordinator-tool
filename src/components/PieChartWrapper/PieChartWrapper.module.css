.chartContainer {
    position: relative
}

.info {
    position: absolute;
    text-align: center;			
    width: 150px;					
    padding: 10px;				
    font-size: 13px;		
    border: 0px;		
    border-radius: 8px;			
    pointer-events: none;	
    opacity: 0;
    background-color: #d5e2e8;
    left: 20%;
    bottom: 0;
    font-weight: 700;
    text-transform: uppercase;
    color: #5c5d5e;
}
