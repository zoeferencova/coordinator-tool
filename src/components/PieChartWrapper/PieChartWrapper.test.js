import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PieChartWrapper from './PieChartWrapper'

describe(`PieChartWrapper component`, () => {
  it('renders a .PieChartWrapper by default', () => {
    const wrapper = shallow(<PieChartWrapper />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})