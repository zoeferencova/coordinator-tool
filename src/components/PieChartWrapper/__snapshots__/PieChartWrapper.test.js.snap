// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PieChartWrapper component renders a .PieChartWrapper by default 1`] = `
<div
  className="chartContainer"
>
  <div />
  <div
    className="info"
  />
</div>
`;
